applications/nrf5340_audio/unicast_server/README.rst

```diff
@@ -0,0 +1,147 @@
+
+   **LED3** starts blinking green on every device to indicate the ongoing CPU activity on the application core.
+#. Wait for the **LED1** on the gateway to start blinking blue.
+   This happens shortly after programming the development kit and indicates that the gateway device is connected to at least one headset and ready to send data.
+#. Search the list of audio devices listed in the sound settings of your operating system for *nRF5340 USB Audio* (gateway) and select it as the output device.
+#. Connect headphones to the **HEADPHONE** audio jack on both headset devices.
+#. Start audio playback on your PC from any source.
+#. Wait for **LED1** to blink blue on the headset.
+   When they do, the audio stream has started on the headset.
+
+   .. note::
+      The audio outputs only to the left channel of the audio jack, even if the given headset is configured as the right headset.
+      This is because of the mono hardware codec chip used on the development kits.
+      If you want to play stereo sound using one development kit, you must connect an external hardware codec chip that supports stereo.
+
+#. Wait for **LED2** to light up solid green on the headsets to indicate that the audio synchronization is achieved.
+#. Press the **VOL+** button on one of the headsets.
+   The playback volume increases for the headset.
+#. If you use more than one headset, hold down the **VOL+** button and press the **RESET** button on a headset.
+   After startup, this headset will be configured as the right channel headset.
+#. If you use more than one headset, hold down the **VOL-** button and press the **RESET** button on a headset.
+   After startup, this headset will be configured as the left channel headset.
+   You can also just press the **RESET** button to restore the original programmed settings.
+
+For other testing options, refer to :ref:`nrf53_audio_unicast_server_app_ui`.
+
+After the kits have paired for the first time, they are now bonded.
+This means the Long-Term Key (LTK) is stored on each side, and that the kits will only connect to each other unless the bonding information is cleared.
+To clear the bonding information, press and hold **BTN 5** during boot or reprogram all the development kits.
+
+When you finish testing, power off the nRF5340 Audio development kits by switching the power switch from On to Off.
+
+.. _nrf53_audio_unicast_server_app_testing_steps_cis_walkie_talkie:
+
+Testing the walkie-talkie demo
+==============================
+
+Testing the walkie-talkie demo is identical to the default testing procedure, except for the following differences:
+
+* You must enable the Kconfig option mentioned in :ref:`nrf53_audio_app_configuration_enable_walkie_talkie` before building the application.
+* Instead of controlling the playback, you can speak through the PDM microphones.
+  The line is open all the time, no need to press any buttons to talk, but the volume control works as in the default testing procedure.
```

Comment:
This section is just copy-paste from unicast_client, right? Any way to link it in? That way we don't have to maintain two copies of the same text

Reply:
I'll do this in the next iteration.